### PR TITLE
Aa #8 information button

### PR DESCRIPTION
**Pull request by Amele-A**
Add navigation to Index- button on Main page. when the button click it takes to the index-page.

### CODE REVIEW

File: screens/search_screen/search.tsx [hunk 1]
```diff
@@ -31,6 +32,19 @@ const Search: FC<SearchScreenProps> = (props) => {
...
+
+  const submitHandler = () => {
+    if (city === "") {
+      Alert.alert("Can't leave blank. Please enter a city.")
+    } else {
+      return setCity(city);
+    }
+  }
+
+  console.log(data);
```
Comment (kmausolf) [inline screens/search_screen/search.tsx]: Leftover console log here.

**Review by lindakw — APPROVED** [review]
Everything looks good!

**Review by kmausolf — CHANGES_REQUESTED** [review]
I think you want to change the base branch to `development` here instead of `main`.  Also, there's a bit of overlap between your PR and Linda's from earlier.  When hers is merged, pull from `development` and push.  I'll remove the hold when the base branch is set to `development` to ensure code isn't being merged to the wrong place.

**Comment by kmausolf** [comment]
Other than what was mentioned, the PR looks great!

**Comment by kmausolf** [comment]
Thanks @lindakw !  Just waiting on the pull from development.

**Review by kmausolf — APPROVED** [review]

**Review by Kacique — COMMENTED** [review]
All good

**Pull request merged**